docs/reference/composites.rst

```diff
@@ -15,11 +15,12 @@ for using the D-Wave system.
 CutOffs
 =======
 
+Downsize the binary quadratic model (BQM) submitted to the child sampler by retaining
```

Comment:
think you want to make this prune as well...

Reply:
Thanks